src/kernel-install/install.conf

```diff
@@ -0,0 +1,15 @@
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it under the
+#  terms of the GNU Lesser General Public License as published by the Free
+#  Software Foundation; either version 2.1 of the License, or (at your option)
+#  any later version.
+#
+# Entries in this file show the compile time defaults. Local configuration
+# should be created by either modifying this file, or by creating "drop-ins" in
+# the sleep.conf.d/ subdirectory. The latter is generally recommended.
+# Defaults can be restored by simply deleting this file and all drop-ins.
+#
+# See kernel-install(8) for details.
+
+#KERNEL_INSTALL_FLAVOUR=legacy|BLS-EFI|BLS-unified|BLS-EBLP
```

Comment:
layout=legacy|bls-efi|... ?

(We don't need to namespace the configuration option with KERNEL_INSTALL_ here, the file is just for us.)